### DISCUSSION TITLE
DNS Challenge with Adguard Home?

### DISCUSSION THREAD

**Discussion by maurxce** [NginxProxyManager/nginx-proxy-manager#3586] [Q&A]
Hey! I just recently set up Adguard Home as a selfhosted DNS.
I'm using a custom (invalid) tld for my services and would like to use NPM to get a SSL Certificate.

Are there any docs on this? I'm unsure as to what I should select for the DNS provider
![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/91284394/3a0561d8-651c-4ad9-82be-a3c5786643ef)

**Comment by Jacksaur** [comment]
Were you able to make any progress on this? Considering the same myself.

**Reply by maurxce** [reply]
No, I switched to a proper tld/domain for my home net
I'd imagine you can probably just use certbot directly to create the certificate and then import it via the web gui

**Reply by Jacksaur** [reply]
Sounds like a good alternative, cheers.
